stage2/04-install-piscsi/00-packages

```diff
@@ -0,0 +1 @@
+git libspdlog-dev vsftpd
```

Comment:
How about we install *all* required package for piscsi here, and then use the new `--skip_packages` parameter with easyinstall.sh to bypass that step in the script? Avoids two roundtrips to the package manager.

Reply:
It does, but then we have two copies of the list to maintain. I'd rather have the image build take a little longer than have two copies of the same info.